diesel/src/pg/query_builder/query_fragment_impls.rs

```diff
@@ -1,11 +1,26 @@
 use pg::Pg;
 use query_builder::{AstPass, QueryFragment};
-use query_builder::for_update_clause::ForUpdateClause;
+use query_builder::for_update_clause::{ForUpdateClause, NoModifier, NoWaitModifier,
+                                       SkipLockedModifier};
 use result::QueryResult;
 
-impl QueryFragment<Pg> for ForUpdateClause {
+impl QueryFragment<Pg> for ForUpdateClause<NoModifier> {
     fn walk_ast(&self, mut out: AstPass<Pg>) -> QueryResult<()> {
         out.push_sql(" FOR UPDATE");
         Ok(())
     }
 }
+
+impl QueryFragment<Pg> for ForUpdateClause<SkipLockedModifier> {
```

Comment:
I'd rather we implement `QueryFragment` for the modifiers rather than duplicate this impl 3 times.

Reply:
Will do